packages/components/psammead-story-promo-list/CHANGELOG.md

```diff
@@ -3,5 +3,6 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 1.0.0-alpha.0 | [PR#679](https://github.com/BBC-News/psammead/pull/679) Bump version number |
```

Comment:
I am not sure if we should bump the alpha components. For example here, it is supposed that `0.1.0-alpha.2` is a pre-release for version `0.1.0` and I would be expecting a bump to 0.1.0 once it is a stable version.

Reply:
https://github.com/bbc/psammead/issues/650#issuecomment-503192541